app/src/main/java/com/project/squirrelobserver/squirrelObserver/RecordActorTabActivity.java

```diff
@@ -222,6 +224,33 @@ public void onResume() {
         }
     }
 
+    public void switchToAllOccurances() {
+        for (int i = 0; i < listRecent.size(); i++) {
+            if(!listRecent.get(i).isEnabled()) {
+                disabledListRecent.add(listRecent.get(i));
+                listRecent.get(i).setEnabled(true);
+            }
+        }
+        for (int i = 0; i < list.size(); i++) {
```

Comment:
Since your disabled button list is just keeping track of pointers to all buttons that were disabled, you don't actually need to use two lists. You can just throw them all in the same list.